Northstar.CustomServers/mod/scripts/vscripts/gamemodes/_gamemode_cp.nut

```diff
@@ -671,12 +687,25 @@ string function CaptureStateToString( int state )
 void function DEV_PrintHardpointsInfo()
 {
 	foreach (entity hardpoint in HARDPOINTS)
-	{
-		printt(
+	{	
```

Comment:
Something like this should look cleaner

```suggestion
	{
		string hardpointGroup
		// update to the method i mentioned in the previous comment
		if((GetMapName()=="mp_homestead")&&(!hardpoint.HasKey("hardpointGroup")))
			hardpointGroup = "B"
		else
			hardpoint.kv.hardpointGroup
		printt(
			"Hardpoint:", hardpointGroup,
			"|Team:", Dev_TeamIDToString(hardpoint.GetTeam()),
			"|State:", CaptureStateToString(hardpoint.GetHardpointState()),
			"|Progress:", GetGlobalNetFloat("objective" + hardpointGroup + "Progress")
		)
	}
```

Reply:
with the encapsulation this is much more readable now